test/size.txt

```diff
@@ -1,98 +1,98 @@
       max   |         min   |        gzip   |   file
       ---   |         ---   |         ---   |   ---
-   290 kB   |    31.63 kB   |    12.49 kB   |   alp.js / alp.max.js
```

Comment:
@jridgewell seems to be some heft increases on the babel sizes, lmk if theres any option that i missed turning on/off that would have done this.

it doesnt really affect the files we serve built from CC.

Reply:
This is because of `babel-polyfill` which is **MASSIVE**